WorkflowFonctionAdminBundle/Controller/Admin/WorkflowFonctionController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFonctionAdminBundle\Controller\Admin;
+
+use OpenOrchestra\BackofficeBundle\Controller\AbstractAdminController;
+use OpenOrchestra\WorkflowFonction\Event\WorkflowFonctionEvent;
+use OpenOrchestra\WorkflowFonction\WorkflowFonctionEvents;
+use OpenOrchestra\WorkflowFonction\Model\WorkflowFonctionInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as Config;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class WorkflowFonctionController
+ */
+class WorkflowFonctionController extends AbstractAdminController
+{
+    /**
+     * @param Request $request
+     * @param string  $workflowFonctionId
+     *
+     * @Config\Route("/workflowfonction/form/{workflowFonctionId}", name="open_orchestra_backoffice_workflowfonction_form")
```

Comment:
you can put workflowfonction as a prefix for all the route of the class, See RoleController for an exemple